guides/doc-Managing_Content/master.adoc

```diff
@@ -25,12 +25,14 @@ include::common/assembly_managing-alternate-content-sources.adoc[leveloffset=+1]
 
 include::common/assembly_importing-content.adoc[leveloffset=+1]
 
+include::common/assembly_restricting-hosts-access-to-content.adoc[leveloffset=+1]
+
+include::common/assembly_synchronizing-content-between-servers.adoc[leveloffset=+1]
+
 include::common/assembly_managing-application-life-cycles.adoc[leveloffset=+1]
 
 include::common/assembly_managing-content-views.adoc[leveloffset=+1]
 
-include::common/assembly_synchronizing-content-between-servers.adoc[leveloffset=+1]
-
```

Comment:
Is this necessary?

Reply:
Definitely not necessary. Not sure how or why I ended up moving that.